native-image-tests/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+  id "com.palantir.graal" version "0.7.1"
+}
+
+dependencies {
+  implementation deps.assertj
+  implementation "org.junit.jupiter:junit-jupiter-api:5.7.0"
+  implementation "org.junit.jupiter:junit-jupiter-engine:5.7.0"
+  implementation "org.junit.platform:junit-platform-console:1.7.0"
+
+  compileOnly deps.jsr305
+}
+
+animalsniffer {
+  ignoreFailures = true
+}
+
+def graalHome = "/Library/Java/JavaVirtualMachines/graalvm-ce-java11-20.2.0/Contents/Home"
```

Comment:
It would be nice to check if the palantir plugin has already populated ~/.gradle/caches/com.palantir.graal/20.2.0/11/graalvm-ce-java11-20.2.0

Reply:
Just read "Seed using the Graal from the Graal Gradle plugin, not the local machine"